### ISSUE TITLE
'unsafe-inline' is ignored if either a hash or nonce value is present in the source list

### ISSUE THREAD

**Issue by ace37** [juunas11/aspnetcore-security-headers#31]
I got error while using this lib

![image](https://user-images.githubusercontent.com/16698681/46400260-a3fe1580-c702-11e8-93ca-ec243eb401d1.png)

**Comment by juunas11** [comment]
That's not really related to this library, but how you are using Content Security Policy. You can't use inline styles if you specify a nonce.

**Comment by ace37** [comment]
app.UseCsp(csp =>
            {
                csp.AllowScripts
                    .FromSelf()
                    .From("www.google.com")
                    .AllowUnsafeInline()
                    .AddNonce();
                csp.AllowStyles
                    .FromSelf()
                    .From("www.google.com")
                    .AllowUnsafeInline()
                    .AddNonce();
            });

**Comment by ace37** [comment]
i dont use nonce on lib with inline style

**Comment by juunas11** [comment]
But you have added a nonce to the CSP. Your browser is ignoring unsafe-inline since a nonce is in the CSP.

**Comment by ace37** [comment]
how i can resolve it?

**Comment by ace37** [comment]
Is it ok when i see empty nonce?

![image](https://user-images.githubusercontent.com/16698681/46400897-2affbd80-c704-11e8-89b0-5e05b9f7cf04.png)

**Comment by rhux** [comment]
Yes I'm seeing that issue too ... the helper tag isn't generating a nonce value.

**Comment by DeeAjith** [comment]
nonce will not be visible in inspect element if you rightclick the page and click on view source code there you can see the nonce values
